src/requirements.txt

```diff
@@ -1,7 +1,7 @@
 requests==2.32.4
-google-adk==1.5.0
+google-adk==1.10.0
 google-generativeai==0.8.5
-litellm==1.70.0
+litellm==1.75.4
 boto3==1.38.19
```

Comment:
nice to see some pretty big jumps in versions, hopefully this gives us some improvement under the cover!